reference/enterprise-api-ref/inventory.markdown

```diff
@@ -0,0 +1,255 @@
+                                                                                                                                     "Host name",
+                                                                                                                                     "OS type",
+                                                                                                                                     "IPv4 addresses",
+                                                                                                                                     "CFEngine version",
+                                                                                                                                     "Ports listening"
+                                                                                                                                  ],
+                                                                                                                                  "hostContextExclude":[  
+                                                                                                                                     "policy_server"
+                                                                                                                                  ]
+                                                                                                                               }
+'
+```
+
+**Example Request Body:**
+
+
+```
+{  
+   "sort":"Host name",
+   "filter":{  
+      "Host name":{  
+         "l":"%value1%",
+         "nil":"%value2%"
+      }
+   },
+   "select":[  
+      "Host name",
+      "OS type",
+      "IPv4 addresses",
+      "CFEngine version",
+      "Ports listening"
+   ],
+   "hostContextExclude":[  
+      "policy_server"
+   ]
+}
+```
+
+**Example response:**
+
+
+```
+{
+  "data": [
+    {
+      "header": [
+        {
+          "columnName": "Host name",
+          "columnType": "STRING"
+        },
+        {
+          "columnName": "OS type",
+          "columnType": "STRING"
+        },
+        {
+          "columnName": "IPv4 addresses",
+          "columnType": "STRING"
+        },
+        {
+          "columnName": "CFEngine version",
+          "columnType": "STRING"
+        },
+        {
+          "columnName": "Ports listening",
+          "columnType": "STRING"
+        }
+      ],
+      "queryTimeMs": 30,
+      "rowCount": 2,
+      "rows": [
+        [
+          "host1.cfengine.com",
+          "linux",
+          "128.30.23.33",
+          "3.10.0",
+          "22, 25, 80, 443, 5308, 9000"
+        ],
+        [
+          "host2.cfengine.com",
+          "linux",
+          "184.45.1.75",
+          "3.10.0",
+          null
+        ]
+      ]
+    }
+  ],
+  "meta": {
+    "count": 1,
+    "page": 1,
+    "timestamp": 1496222472,
+    "total": 1
+  }
+}
+```
+
+## List of variables
+
+**URI:** https://hub.cfengine.com/api/variables-dictionary
```

Comment:
@maciejmrowiec @michaelclelland is this a good API endpoint? We report on variables that are not part of inventory, this would be a bit misleading to me. By looking only at the api path I would assume this is where I would query on arbitrary variables not limited to those tagged for inventory.

Reply:
As far as I understand I have to change endpoint to https://hub.cfengine.com/api/inventory/variables-dictionary ?

Reply:
I believe it should be called attributes-dictionary

Reply:
Still looks like the docs reference variables-dictionary instead if the new name